samples/start-app/php-symfony/src/Controller/HomepageController.php

```diff
@@ -4,29 +4,67 @@
 
 namespace App\Controller;
 
+use App\Entity\Exception\UserNotFoundException;
 use App\Repository\TokenRepositoryInterface;
+use App\Repository\UserRepositoryInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Cookie;
+use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
 final class HomepageController extends AbstractController
 {
     public function __construct(
         private readonly TokenRepositoryInterface $tokenRepository,
-        private readonly string                   $projectDir
+        private readonly UserRepositoryInterface  $userRepository,
+        private readonly string                   $projectDir,
+        private readonly string                   $openIdAuthentication
     )
     {
     }
 
     #[Route('/', name: 'homepage', methods: ['GET'])]
-    public function __invoke(): Response
+    public function __invoke(Request $request): Response
     {
-        if (!$this->tokenRepository->hasToken()) {
-            return new Response(file_get_contents($this->projectDir . '/templates/no_access_token.html'));
+        try {
+            $divToReplace = "<div>UserInformation</div>";
+            $divUserInformation = "";
+
+            if (!$this->tokenRepository->hasToken()) {
+                return new Response(
+                    str_replace(
```

Comment:
I think you dont need to replace here, juste delete <div>UserInformation</div> in template

Reply:
I would not do that as the error template could be used in another use case, where cookies are correctly filled and so the user would be connected.